src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -84,11 +84,6 @@ public function testUnsilencing()
 
     public function testStacking()
```

Comment:
This test case doesn't test anything anymore.
In fact, this makes me think we should deprecate `ErrorHandler::stackErrors()` / `ErrorHandler::unstackErrors()` in 3.4
they are workaround for bugs that are fixed in 5.5.9
this would make this test case `@group legacy` thus the removal would be plain legitimate.
would you mind doing so on 3.4 ?